packages/kolme-cli/Cargo.toml

```diff
@@ -22,6 +23,7 @@ serde = { workspace = true }
 serde_json = { workspace = true }
 tokio = { workspace = true }
 tracing = { workspace = true }
+version-compare = "0.2.0"
```

Comment:
Oh, interesting. From a quick review of the crate, seems like a reasonable set of assumptions.